src/chainstate/burn/db/sortdb.rs

```diff
@@ -6909,20 +6998,14 @@ pub mod tests {
         // accept blocks 5 and 7 in one fork, and 6, 8, 9 in another.
         // Stacks fork 1,2,3,4,5,7 will be the longest fork.
         // Stacks fork 1,2,3,4 will overtake it when blocks 6,8,9 are processed.
-        let mut parent_stacks_block_hash = BlockHeaderHash([0x04; 32]);
-        for (i, height) in [5, 7].iter().zip([5, 6].iter()) {
+        for (i, height) in [7].iter().zip([6].iter()) {
```

Comment:
does this loop still match the comment just above?

Reply:
yes; it was a bug to use i=5 and height=5